From 7d22b36145265daecc7d9bcf69fd675415171129 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Sun, 1 May 2016 07:02:53 -0400
Subject: [PATCH 3/6] Remove strip step in Makefile

---
 Makefile.ffmpeg | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index a7ddd5f..6910558 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -241,7 +241,9 @@ configure:
 			--disable-decoder=idf \
 			--disable-decoder=hevc \
 			--enable-decoder=opus \
-			--cross-prefix=$(HOST)-
+			--cross-prefix=$(HOST)- \
+			--disable-stripping
+
 
 .PHONY : clean
 clean:
@@ -254,5 +256,3 @@ checkout:
 .PHONY : install
 install:
 	cd ffmpeg; make -j9 DESTDIR="$(WORK)/ffmpeg_compiled" install
-	$(HOST)-strip ffmpeg_compiled/usr/lib/*.so
-
-- 
2.5.0

