From df9e67aa6cbb1e4c2a420ddf22d19eb7e6c99fd6 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Mon, 25 Jun 2018 09:21:34 -0400
Subject: [PATCH] Remove strip step in Makefile

---
 Makefile.ffmpeg | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index d712f6c..5f25c26 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -241,7 +241,9 @@ configure:
 			--disable-decoder=idf \
 			--disable-decoder=hevc \
 			--enable-decoder=opus \
-			--cross-prefix=$(HOST)-
+			--cross-prefix=$(HOST)- \
+			--disable-stripping
+
 
 .PHONY : clean
 clean:
@@ -254,5 +256,3 @@ checkout:
 .PHONY : install
 install:
 	cd ffmpeg; make -j9 DESTDIR="$(WORK)/ffmpeg_compiled" install
-	$(HOST)-strip ffmpeg_compiled/usr/lib/*.so
-
-- 
2.7.4

