From 0e0d050cb77a0e3270c17191ad9c918390efbf8d Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Mon, 25 Jun 2018 09:08:35 -0400
Subject: [PATCH] fix tar command with DIST

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cb59249..929f0e0 100644
--- a/Makefile
+++ b/Makefile
@@ -89,4 +89,4 @@ dist: omxplayer.bin omxplayer.1
 	cp README.md $(DIST)/usr/share/doc/omxplayer/README
 	cp omxplayer.1 $(DIST)/usr/share/man/man1
 	cp -P ffmpeg_compiled/usr/lib/*.so* $(DIST)/usr/lib/omxplayer/
-	cd $(DIST); tar -czf ../$(DIST).tgz *
+	tar -czf omxplayer-dist.tar.gz $(DIST)
-- 
2.7.4

